src/Extensions/Hosting.Services/IServiceContextAccessor.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services
+{
+	/// <summary>
+	/// Provides access to service fabric service contest
+	/// </summary>
+	public interface IServiceContextAccessor<out TServiceContext>
+		where TServiceContext : ServiceContext
+	{
+		/// <summary>
+		/// Get service context if it's available
+		/// </summary>
+		TServiceContext? ServiceContext { get; }
+
```

Comment:
Are we including two lines between methods in this changelist?

Reply:
Unfortunately, it's still two lines :( Fixed, thank you